tests/components/velbus/conftest.py

```diff
@@ -96,7 +96,7 @@ def mock_module_subdevices() -> AsyncMock:
     """Mock a velbus module."""
     module = AsyncMock(spec=Module)
     module.get_type_name.return_value = "VMB2BLE"
-    module.get_addresses.return_value = [99]
+    module.get_addresses.return_value = [88]
```

Comment:
Is this representative of the actual address values. This looks very much non unique. So if two config entries exist they could start overwriting each others devices.

Reply:
Ps. This should not be fixed in this pull if it is an issue. Better done separate